### PR TITLE
[zh] Sync #14491 Ambient upgrade doc into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14491 Ambient upgrade doc into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: 使用 Helm 升级
+description: 如何使用 Helm 升级 Ambient Mesh。
+weight: 5
+owner: istio/wg-environments-maintainers
+test: yes
+status: Experimental
+---
+
+根据本指南可以使用 [Helm](https://helm.sh/docs/) 升级和配置 Ambient Mesh。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:10]:
```suggestion
根据本指南使用 [Helm](https://helm.sh/docs/) 升级和配置 Ambient Mesh。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+本指南假设您已经使用之前的 Istio 次要版本或补丁版本执行了
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:11]:
```suggestion
本指南假设您已经使用之前的 Istio 次要版本或补丁版本
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+Ambient Mesh [使用 Helm 安装](/zh/docs/ops/ambient/install/helm-installation/)。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:12]:
```suggestion
[通过 Helm 安装](/zh/docs/ops/ambient/install/helm-installation/)了 Ambient Mesh。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+{{< boilerplate ambient-alpha-warning >}}
+
+{{< warning >}}
+与 Sidecar 模式相反，`Ambient` 支持将应用程序 Pod 移动到已经升级的数据平面，
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:17]:
```suggestion
与 Sidecar 模式相比，`Ambient` 支持将应用程序 Pod 移动到已升级的数据平面，
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+而无需强制重新启动或重新编排正在运行的应用程序 Pod。
+但是，升级数据平面**将**短暂中断被升级节点上的所有工作负载流量，
+并且 Ambient 当前不支持数据平面的金丝雀升级。
+
+建议使用节点封锁和蓝/绿节点池来控制生产环境升级期间应用程序 Pod 流量中断的爆炸半径。
+有关详细信息，请参阅 Kubernetes 提供商文档。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:23]:
```suggestion
有关详细信息，请参阅您的 Kubernetes 提供商文档。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+{{< /warning >}}
+
+## 先决条件 {#prerequisites}
+
+1. 按照[使用 Helm 安装](/zh/docs/ops/ambient/install/helm-installation/)并满足该指南中的所有先决条件，通过 Helm 安装 Ambient Mesh。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:28]:
```suggestion
1. 按照[使用 Helm 安装](/zh/docs/ops/ambient/install/helm-installation/)并满足该指南中的所有先决条件，
   通过 Helm 安装 Ambient Mesh。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+1. 更新 Helm 仓库：
+
+    {{< text syntax=bash snip_id=update_helm >}}
+    $ helm repo update istio
+    {{< /text >}}
+
+## 就地升级 {#in-place-upgrade}
+
+您可以使用 Helm 升级工作流程在集群中执行 Istio 就地升级。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:38]:
```suggestion
您可以使用 Helm 升级工作流程在集群中就地升级 Istio。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+在升级 Istio 之前，建议运行 `istioctl x precheck` 命令以确保升级与您的环境兼容。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:40]:
```suggestion
在升级 Istio 之前，建议先运行 `istioctl x precheck` 命令以确保升级与您的环境兼容。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+### 手动升级 CRD 和 Istio Base Chart {#manually-upgrade-the-crds-and-istio-base-chart}
+
+1. 升级 Kubernetes 自定义资源定义 ({{< gloss >}}CRDs{{</ gloss >}})：
+
+    {{< text syntax=bash snip_id=manual_crd_upgrade >}}
+    $ kubectl apply -f manifests/charts/base/crds
+    {{< /text >}}
+
+1. 升级 Istio base Chart：
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:61]:
```suggestion
1. 升级 Istio Base Chart：
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+    {{< text syntax=bash snip_id=upgrade_base >}}
+    $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
+    {{< /text >}}
+
+### 升级 Istio discovery 组件 {#upgrade-the-istio-discovery-component}
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:67]:
```suggestion
### 升级 Istio Discovery 组件 {#upgrade-the-istio-discovery-component}
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+
+Istiod 是控制平面组件，用于管理和配置代理以在 Ambient Mesh 内路由流量。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:69]:
```suggestion
Istiod 是管理和配置代理的控制平面组件，在 Ambient Mesh 内路由流量。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+{{< /text >}}
+
+### 升级 ztunnel 组件 {#upgrade-the-ztunnel-component}
+
+ztunnel DaemonSet 是 Ambient 中的 L4 节点代理组件。
+
+{{< warning >}}
+就地升级 ztunnel 将短暂中断节点上的所有 Ambient Mesh 流量。
+建议使用节点封锁和蓝/绿节点池来减轻生产环境升级期间的爆炸半径风险。
+有关详细信息，请参阅 Kubernetes 提供商文档。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:82]:
```suggestion
有关详细信息，请参阅您的 Kubernetes 提供商文档。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
...
+
+{{< text syntax=bash snip_id=upgrade_ztunnel >}}
+$ helm upgrade ztunnel istio/ztunnel -n istio-system
+{{< /text >}}
+
+### 升级 CNI 组件 {#upgrade-the-cni-component}
+
+Istio CNI 代理负责检测属于 Ambient Mesh 的 Pod，
+并配置 Pod 和 ztunnel DaemonSet 之间的流量重定向。
+它不是数据平面或控制平面的一部分。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:93]:
```suggestion
Istio CNI Agent 不是数据平面或控制平面的一部分。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 14]
```diff
@@ -0,0 +1,143 @@
...
+有关详细信息，请参阅 Kubernetes 提供商文档。
+{{< /warning >}}
+
+{{< text syntax=bash snip_id=upgrade_ztunnel >}}
+$ helm upgrade ztunnel istio/ztunnel -n istio-system
+{{< /text >}}
+
+### 升级 CNI 组件 {#upgrade-the-cni-component}
+
+Istio CNI 代理负责检测属于 Ambient Mesh 的 Pod，
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:91]:
```suggestion
Istio CNI Agent 负责检测属于 Ambient Mesh 的 Pod，
```
下面几个 agent 也没翻

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+并配置 Pod 和 ztunnel DaemonSet 之间的流量重定向。
+它不是数据平面或控制平面的一部分。
+
+Istio CNI Agent 1.x 版本兼容控制平面 1.x-1、1.x 以及 1.x+1 版本，
+这意味着 Istio CNI Agent 和 Istio 控制平面能够以任何先后顺序独立升级。
+只要它们的版本差异在一个小版本之内。
+
+{{< warning >}}
+升级 Istio CNI 代理将重新配置节点上的网络，因此会暂时中断节点流量。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:100]:
```suggestion
升级 Istio CNI Agent 将重新配置节点上的网络，因此会暂时中断节点流量。
```
同上，需要一致性

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 16]
```diff
@@ -0,0 +1,143 @@
...
+为了在 Istio CNI 代理升级期间管理应用程序 Pod 的爆炸半径，建议使用节点警戒线。
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:101]:
```suggestion
为了在 Istio CNI Agent 升级期间管理应用程序 Pod 的爆炸半径，建议使用节点警戒线。
```

**Review by windsonsea — COMMENTED** [review]
Raise some text suggestions for discussion.

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+{{< /text >}}
+
+{{< warning >}}
+[Helm 在执行升级时不会升级或删除 CRD](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations)。
+由于此限制，在使用 Helm 升级 Istio 时需要执行额外的步骤。
+{{< /warning >}}
+
+### 手动升级 CRD 和 Istio Base Chart {#manually-upgrade-the-crds-and-istio-base-chart}
+
+1. 升级 Kubernetes 自定义资源定义 ({{< gloss >}}CRDs{{</ gloss >}})：
```
Comment (windsonsea) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:55]:
```suggestion
1. 升级 Kubernetes 自定义资源定义 ({{< gloss "crd" >}}CRD{{</ gloss >}})：
```
这里有 lint 错误，试试这样呢

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 18]
```diff
@@ -0,0 +1,144 @@
...
+title: 使用 Helm 升级
+description: 如何使用 Helm 升级 Ambient Mesh。
+weight: 5
+owner: istio/wg-environments-maintainers
+test: yes
+status: Experimental
+---
+
+根据本指南使用 [Helm](https://helm.sh/docs/) 升级和配置 Ambient Mesh。
+本指南假设您已经使用之前的 Istio 次要版本或补丁版本
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:11]:
```suggestion
本指南假设您已经[通过 Helm 安装](/zh/docs/ops/ambient/install/helm-installation/)了
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 19]
```diff
@@ -0,0 +1,144 @@
...
+[通过 Helm 安装](/zh/docs/ops/ambient/install/helm-installation/)了 Ambient Mesh。
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:12]:
```suggestion
Istio 的一个较早的次要版本或补丁版本的 Ambient Mesh。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 20]
```diff
@@ -0,0 +1,144 @@
...
+
+1. 升级 Istio Base Chart：
+
+    {{< text syntax=bash snip_id=upgrade_base >}}
+    $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
+    {{< /text >}}
+
+### 升级 Istio Discovery 组件 {#upgrade-the-istio-discovery-component}
+
+Istiod 是管理和配置代理的控制平面组件，在 Ambient Mesh 内路由流量。
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:70]:
```suggestion
Istiod 是管理和配置代理以在 ambient mesh 中路由流量的控制平面组件。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 21]
```diff
@@ -0,0 +1,144 @@
...
+并配置 Pod 和 ztunnel DaemonSet 之间的流量重定向。
+Istio CNI Agent 不是数据平面或控制平面的一部分。
+
+Istio CNI Agent 1.x 版本兼容控制平面 1.x-1、1.x 以及 1.x+1 版本，
+这意味着 Istio CNI Agent 和 Istio 控制平面能够以任何先后顺序独立升级。
+只要它们的版本差异在一个小版本之内。
+
+{{< warning >}}
+升级 Istio CNI Agent 将重新配置节点上的网络，因此会暂时中断节点流量。
+为了在 Istio CNI Agent 升级期间管理应用程序 Pod 的爆炸半径，建议使用节点警戒线。
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:102]:
```suggestion
为了控制 Istio CNI Agent 升级期间影响应用程序 Pod 的范围，建议使用节点警戒线。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 22]
```diff
@@ -0,0 +1,144 @@
...
+
+{{< boilerplate ambient-alpha-warning >}}
+
+{{< warning >}}
+与 Sidecar 模式相比，`Ambient` 支持将应用程序 Pod 移动到已升级的数据平面，
+而无需强制重新启动或重新编排正在运行的应用程序 Pod。
+但是，升级数据平面**将**短暂中断被升级节点上的所有工作负载流量，
+并且 Ambient 当前不支持数据平面的金丝雀升级。
+
+建议使用节点封锁和蓝/绿节点池来控制生产环境升级期间应用程序 Pod 流量中断的爆炸半径。
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:22]:
```suggestion
建议使用节点封锁和蓝/绿节点池来控制生产环境升级期间应用程序 Pod 流量中断的影响范围。
```

File: content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md [hunk 23]
```diff
@@ -0,0 +1,144 @@
...
+$ helm upgrade istiod istio/istiod -n istio-system
+{{< /text >}}
+
+### 升级 ztunnel 组件 {#upgrade-the-ztunnel-component}
+
+ztunnel DaemonSet 是 Ambient 中的 L4 节点代理组件。
+
+{{< warning >}}
+就地升级 ztunnel 将短暂中断节点上的所有 Ambient Mesh 流量。
+建议使用节点封锁和蓝/绿节点池来减轻生产环境升级期间的爆炸半径风险。
```
Comment (hanxiaop) [inline content/zh/docs/ops/ambient/upgrade/helm-upgrade/index.md:82]:
```suggestion
建议使用节点封锁和蓝/绿节点池来减轻生产环境升级期间的影响范围。
```

**Review by windsonsea — APPROVED** [review]
thanks, lgtm

**Pull request merged**